src/common/actions/register-name.js

```diff
@@ -9,16 +9,26 @@ import { requestBuilds } from './snap-builds';
 
 const BASE_URL = conf.get('BASE_URL');
 
+// action types
 export const REGISTER_NAME = 'REGISTER_NAME';
 export const REGISTER_NAME_SUCCESS = 'REGISTER_NAME_SUCCESS';
 export const REGISTER_NAME_ERROR = 'REGISTER_NAME_ERROR';
 export const REGISTER_NAME_CLEAR = 'REGISTER_NAME_CLEAR';
 
+export const CHECK_NAME_OWNERSHIP = 'CHECK_NAME_OWNERSHIP';
```

Comment:
I've been following a convention of using `REQUEST`, `SUCCESS` and `ERROR` suffixes, matching the [Redux real world example](https://github.com/reactjs/redux/blob/master/examples/real-world/src/actions/index.js#L36).